tests/src/e2e.spec.ts

```diff
@@ -1,21 +1,32 @@
 import type { BrowserWindow } from 'electron';
-import type { ElectronApplication, JSHandle, Page } from 'playwright';
+import type { ElectronApplication, JSHandle, Locator, Page } from 'playwright';
 import { _electron as electron } from 'playwright';
 import { afterAll, beforeAll, expect, test } from 'vitest';
 import { expect as playExpect } from '@playwright/test';
-import { existsSync } from 'node:fs';
+import { existsSync, readFileSync } from 'node:fs';
 import { rm } from 'node:fs/promises';
+import * as os from 'node:os'
+import * as path from 'node:path';
 
 let electronApp: ElectronApplication;
 
 let page: Page;
+let userHome = os.homedir();
+const navBarItems = ['Dashboard', 'Containers', 'Images', 'Pods', 'Volumes', 'Settings'];
 
 beforeAll(async () => {
   // remove all videos/screenshots
   if (existsSync('tests/output')) {
     console.log('Cleaning up output folder...');
     await rm('tests/output', { recursive: true, force: true });
   }
+  
+  const settingsPath = path.join(userHome, '.local', 'share', 'containers', 'podman-desktop', 'configuration', 'settings.json');
+  // clean up settings to show initial welcome screen
+  if (existsSync(settingsPath)) {
+    console.log('Removing settings.json to get initial state');
+    await rm(settingsPath, {});
+  }
```

Comment:
Agree

Reply:
That is a good catch. 

There could be a workaround, backing up local settings in beforeall, and putting it back in afterall. Of course, in case of crash or interrupted tests, this has no effect and problem persists.

How about a flag passed to the application to use custom settings file?

Reply:
Thinking more about this... At some point, there will be tests (e2e) that will eventually be messing up with local system. We will not want them to be part of standard developer workflow - develop + run unit tests (+some smokes). They will be cleaning up with podman machines, download tons of data (crc bundles, etc.). So we will eventually end up with either tests in codebase standing out of standard npm tasks, or individual repository...